analyzers/tests/SonarAnalyzer.UnitTest/Common/ReadMeTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.IO;
+using System.Text.RegularExpressions;
+using SonarAnalyzer.Common;
+using SonarAnalyzer.Utilities;
+
+namespace SonarAnalyzer.UnitTest.Common;
+
+[TestClass]
+public class ReadMeTest
+{
+    [TestMethod]
+    public void MentionsTheRightNumbers()
+    {
+        var cs = RuleFinder.GetAnalyzerTypes(AnalyzerLanguage.CSharp).Count();
+        var vb = RuleFinder.GetAnalyzerTypes(AnalyzerLanguage.VisualBasic).Count();
+
+        var match = Regex.Match(Readme(), @"\[(?<cs>[0-9]{3,4})\+ C# rules\].+ and \[(?<vb>[0-9]{3,4})\+ VB\.");
```

Comment:
This should look somehow like this after the split
```suggestion
        var match = Regex.Match(Readme(), $"\[(?<count>\d+)\+ {name} rules\]");
```